src/test/isolation2/expected/concurrent_select_vacuum_ao_partitioned_table.out

```diff
@@ -233,3 +233,4 @@ VACUUM
  1 | 1 | 2 
 (1 row)
 DROP TABLE foo_part;
+DROP
```

Comment:
Unfortunately, if the tests do not pass without DROP, we have to add DROP to the **upstream patch**. I don't know why the tests have passed in upstream, but in this PR each separate commit should bring the whole project to the valid state. So, we have no choice but fixing the tests inside the upstream commit.